src/lib/Libwin/util.c

```diff
@@ -242,7 +242,8 @@ fcntl(int fd, int cmd, long arg)
  * 	just a placeholder for the unix equivalent function.
  *
  * @param[in] path - file path
- * @param[in] resolved_path - resolved path
+ * @param[in] resolved_path - resolved path, if NULL then PATH_MAX bytes will
+ *     be allocated, which the caller of this function is responsible to free
```

Comment:
This is an out.  Also, did you mean 'if not NULL'?

Reply:
No, I mean if equal to NULL. And this parameter is an input. `realpath` will return the allocated memory address as return value.

Reply:
Sorry I was wrong. This is an output. The resolved real path will be stored in this address.